packages/react-scripts/template/README.md

```diff
@@ -18,6 +18,7 @@ You can find the most recent version of this guide [here](https://github.com/fac
 - [Importing a Component](#importing-a-component)
 - [Adding a Stylesheet](#adding-a-stylesheet)
 - [Post-Processing CSS](#post-processing-css)
+- [Adding CSS Preprocessor (SASS, Less etc.](#adding-css-preprocessor-sass-less-etc)
```

Comment:
Nice catch! I think the anchor link [works just fine](https://github.com/tsironis/create-react-app/blob/sass-support-docs/packages/react-scripts/template/README.md#adding-css-preprocessor-sass-less-etc).